test/test_lowlevel.py

```diff
@@ -484,7 +484,7 @@ def test14_templated_arrays(self):
         assert cppyy.gbl.std.vector[cppyy.gbl.std.vector[int]].value_type == 'std::vector<int>'
         assert cppyy.gbl.std.vector['int[1]'].value_type == 'int[1]'
 
-    @mark.xfail
+    @mark.skip
```

Comment:
Why we skip this test?

Reply:
Its an odd one. I only want to skip for osx x86 cling, but couldn't work out how to do it and have xfail for everythig else

Reply:
Not entirely sure which of my changes makes it happen, but on osx x86 cling it crashes the pytest run now.

Reply:
```suggestion
    @mark.xfail(run=not (IS_MAC_X86 and not IS_CLANG_REPL), reason="Crashes on OSX-X86 Cling")
```
The `run` argument of `xfail` can be used to specify that the test should not be run.